NewForce-Cohort-7/tabloidmvc-pack-iron-posse
# Ad branch

Description

Added the ability to view comments and their details regarding to the post.
[Trello Ticket #19][(https://trello.com/c/kg4y9TRL)

Testing Instructions
git add and git commit whatever you're working on on your branch
git checkout main
git fetch --all
git checkout ad-branch

To test this new feature, you will need to have add a test tag. In Visual Studio, right click the TabloidMVC Database in the SQL Server Object Explorer, selecting "Create a New Query."
Next make sure to run the two databases 01_TabloidMVC_Create_DB.sql and 02_TAbloidMVCseed_Data.sql by clicking the same green button on their respective pages.
Type "Select * From Post" to see all the post
To add another, post the following entry:
Insert into Post (Title, Content, ImageLocation, CreateDateTime, PublishDateTime, IsApproved, CategoryId, UserProfileId)
VALUES ('test', 'test', 'test imagelocation', 2020-10-10, 2020-10-10, 1, 1, 1) then run with green button in the left hand corner.
Next you need to make a comment so type Insert into Comment (PostId, UserProfileId, Subject, Content, CreateDateTime)
VALUES (2, 1, 'test', 'test', 2020-10-10)
click the green button again 
Then to view all the comments type SELECT * FROM Comment to make sure it has posted. Then run it with the green button in the left hand corner.
Next Click the execute with debugger solid green button on the program files to launch the app. 
In the browser, please go to Login on the nav bar and log in with this email: [admin@example.com](mailto:admin@example.com)
This should redirect you back to the home page.
Now go into the Post page and click the detail button (looks like an eye) to view the posts details. In there you will see the test comment you made with the sql query.

Checklist:
 My code follows the style guidelines of this project
[x] I have performed a self-review of my code
 [x]I have commented my code, particularly in hard-to-understand areas
 [x]My changes generate no new warnings or errors
 [x]I have added test instructions that prove my fix is effective or that my feature works
